pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -101,6 +101,23 @@ social:
 %br/
 
 .container
+  -# HoC Live Spanish banner
+  - if @language == 'es' || @language == 'la'
+    #hoclivebanner.row
+      %div.banner-box
+        %div.banner-image-box
+          %img.banner-image{src: '/images/hora-del-codigo-en-vivo.png', alt: hoc_s(:hoc_live)}
+        %div.banner-message-box
+          %div.banner-title
+            = hoc_s(:hoc_live_sign_up_title)
+          %div.banner-message
+            = hoc_s(:hoc_live_sign_up_message)
+          %a#signuplivebutton.ctabuttonatag{href: 'http://go.pardot.com/l/153401/2020-09-30/nwvb6j', target: '_blank'}
```

Comment:
Important note: `target: _blank` is a security vulnerability. If we absolutely _must_ open links in new tabs, there are some things we can do to mitigate those security risks, but in this case I don't see a need for us to do that. Situations in which opening a link in a new tab are necessary are mostly those situations in which there are interactions on the page that could be lost if the user unexpectedly navigates away; things like a page with a long form to be filled out or, say, a page with a code editor that may lose progress.

The hourofcode.com homepage is a static page, so we should just leave this anchor as normal.

For more information, see:

- https://medium.com/sedeo/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f
- https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/
- https://blog.dareboost.com/en/2017/03/target-blank-links-rel-noopener-performance-security/

Reply:
I agree that this doesn't have to open on a new tab :)

Reply:
Removed. Thank you for the detailed explanation! It's really helpful.